main.go

```diff
@@ -42,8 +37,18 @@ func main() {
 		c := "Specify the absolute path to the key file. (default: none)"
 		flag.StringVar(&keyFile, "key", "", c)
 	}
+	var printVersion bool
+	{
+		c := "Print current version. (default: false)"
+		flag.BoolVar(&printVersion, "ver", false, c)
```

Comment:
[imo]
Can I get the version by typing likes ```frau --ver```?

I think ```--version``` or ```-v```  option is better.

Reply:
Oh, I think so. I'll fix.

Reply:
fixed!